### DISCUSSION TITLE
Welcome to commit-karma Discussions!

### DISCUSSION THREAD

**Discussion by justinmchase** [justinmchase/commit-karma#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 69robert** [comment]
Carma ?? Am I wrong or Java this all public am I just a joke play with words do i just close it all down ?

**Reply by justinmchase** [reply]
I don't completely follow? I think you're asking if the spelling of "karma" is meant to be "carma"? 

If so, in American English at least, the spelling is [karma](https://www.merriam-webster.com/dictionary/karma).

It also sounds as if you're asking if this is actually running or not, and if so then it is actually installable and running but I haven't really finished it yet so I may suggest holding off for now. If you're willing and interested in helping test it a bit then I think we could probably finish it up and get it live pretty quickly.

Its in the github marketplace but isn't public and hasn't been publicized yet because I am not completely done. I was having trouble finding a cheap serverless database that I could use without incurring a lot of cost.

**Reply by 69robert** [reply]
I am sorry my spelling is not correct .I feel I'm missing something data my accounts banks I have tryed bitcoin 22 hrs closing account can't log on so Manny accounts say linked but wen comes to some only gets my data can't get logged .sorry everthing seems hacked like we see same story but words are changed or hole video has changed I ask people thay have no clue about what is going on sorry if public or just us chatting I never bin in that area before trying to remember the steps to get back I bin trying to figure this out and spelling school was not important life food and more at age 8 life also had food allergies so had to much on a 8 year old since I was 80% burnt at work denied disability took years but ten years ago couldn't Evan right this so if codes are roung or things are my fault I am sorry to everone xo
